cmd/pebble/main.go

```diff
@@ -71,11 +72,26 @@ func main() {
 	wfeImpl := wfe.New(logger, db, va, ca, *strictMode)
 	muxHandler := wfeImpl.Handler()
 
+	if len(c.Pebble.ManagementListenAddress) > 0 {
+		go func() {
+			adminHandler := wfeImpl.ManagementHandler()
+			err = http.ListenAndServeTLS(
+				c.Pebble.ManagementListenAddress,
+				c.Pebble.Certificate,
+				c.Pebble.PrivateKey,
+				adminHandler)
+			cmd.FailOnError(err, "Calling ListenAndServeTLS() for admin interface")
+		}()
+		logger.Printf("Management interface listening on: %s\n", c.Pebble.ManagementListenAddress)
+		logger.Printf("Root CA certificate(s) available at: https://%s%s",
+			c.Pebble.ManagementListenAddress, wfe.RootCertPath)
+	} else {
+		logger.Printf("Management interface is disabled")
```

Comment:
Nit: Since this isn't using a format string you can switch to `logger.Print`
```suggestion
		logger.Print("Management interface is disabled")
```